restcomm/restcomm.http/src/main/java/org/mobicents/servlet/restcomm/http/AccountsEndpoint.java

```diff
@@ -285,6 +337,22 @@ protected Response updateAccount(final String accountSid, final MultivaluedMap<S
                 if ((subject.hasRole("Administrator") && secureLevelControlAccounts(account))
                         || (subject.getPrincipal().equals(accountSid) && subject.isPermitted("RestComm:Modify:Accounts"))) {
                     dao.updateAccount(account);
+
+                    // Update SIP client of the corresponding Account
+                    MultivaluedMap<String, String> clientData = new MultivaluedMapImpl();
+                    String username = data.getFirst("EmailAddress").split("@")[0];
```

Comment:
@gvagenas I think the email address is required when we create new account on Restcomm UI.
Please describe more if I'm wrong. Thanks.